DTS-STN/Scrum-Poker
# Dums 91 owner crown

## [DUMS-91- Add tag for room owner on user list](https://jira-dev.bdm-dev.dts-stn.com/browse/DUMS-91) (Jira Issue)

### Description

This PR adds an identifier for the room host/owner in the shape of a crown. I've also added a tooltip which in its current form is accessible but once we alter the crown to be focusable (for promoting users to owner status), the tooltip will likely have to be changed as well.

### What to test for/How to test

1. Pull in branch
2. Type `npm run dev`
3. Create a room, you'll notice you'll have a crown next to your player name
4. Join the room on a separate tab, you'll notice that the new player does not have a crown, and that the host still has a crown

### Additional Notes

I added the tooltip since I thought it would be a cool addition but I'm not tied to it in any way if the team feels like it should be removed.
